db/migrate/20241016202657_add_can_view_units_permission.rb

```diff
@@ -2,7 +2,6 @@ class AddCanViewUnitsPermission < ActiveRecord::Migration[7.0]
   def up
     ::Hmis::Role.ensure_permissions_exist
     ::Hmis::Role.reset_column_information
-    ::Hmis::Role.where(can_manage_inventory: true).update_all(can_view_units: true, can_manage_units: true)
```

Comment:
I'm suggesting to remove this since we're removing the column `can_manage_inventory`. I remember problems in the past with data in migrations, and the controversy over whether they should be used for schema updates only even though it's sometimes convenient to also use them for data changes, so it just seems safer to me to remove it.

Reply:
That seems fine as long as we are sure that the migration has already been applied everywhere. In this case, it hasn't -- the migration went into release-139 https://github.com/greenriver/hmis-warehouse/pull/4836/files
So we will want to target this PR against 140! And of course be sure that release-139 is deployed independently, before we deploy 140.

Reply:
Alternatively, you can ask if the column exists and only run the migration in that case, that should be safer since we may deploy to different places at different times.

```
::Hmis::Role.where(can_manage_inventory: true).update_all(can_view_units: true, can_manage_units: true) if column_exists?(:hmis_roles, :can_manage_inventory)
```

Reply:
Thanks, good idea!